eng/versioning/version_client.txt

```diff
@@ -103,38 +103,38 @@ com.azure:azure-storage-internal-avro;12.0.1;12.0.2-beta.2
-com.azure.spring:azure-spring-cloud-stream-binder-eventhubs;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-cloud-stream-binder-servicebus-core;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-cloud-stream-binder-servicebus-queue;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-cloud-stream-binder-servicebus-topic;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-cloud-stream-binder-test;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-cloud-telemetry;2.0.0-beta.1;2.0.0
+com.azure.spring:azure-spring-boot-starter-keyvault-secrets;3.0.0;3.2.0-beta.1
+com.azure.spring:azure-spring-boot-starter-servicebus-jms;3.0.0;3.2.0-beta.1
+com.azure.spring:azure-spring-boot-starter-storage;3.0.0;3.2.0-beta.1
+com.azure.spring:azure-spring-boot-starter;3.0.0;3.2.0-beta.1
+com.azure.spring:azure-spring-boot;3.0.0;3.2.0-beta.1
+com.azure.spring:azure-spring-cloud-autoconfigure;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-context;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-messaging;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-cache;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-eventhubs-kafka;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-eventhubs;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-servicebus;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-storage-queue;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-storage;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-stream-binder-eventhubs;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-stream-binder-servicebus-core;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-stream-binder-servicebus-queue;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-stream-binder-servicebus-topic;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-stream-binder-test;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-telemetry;2.0.0;2.2.0-beta.1
 com.azure.spring:azure-spring-data-gremlin;2.3.1-beta.1;2.3.1-beta.1
-com.azure.spring:azure-spring-integration-core;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-integration-eventhubs;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-integration-servicebus;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-integration-storage-queue;2.0.0-beta.1;2.0.0
-com.azure.spring:azure-spring-integration-test;2.0.0-beta.1;2.0.0
+com.azure.spring:azure-spring-integration-core;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-integration-eventhubs;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-integration-servicebus;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-integration-storage-queue;2.0.0;2.2.0-beta.1
+com.azure.spring:azure-spring-integration-test;2.0.0;2.2.0-beta.1
 com.azure.spring:azure-spring-cloud-appconfiguration-config-web;1.2.8-beta.1;1.2.8-beta.1
 com.azure.spring:azure-spring-cloud-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
 com.azure.spring:azure-spring-cloud-feature-management-web;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
1. Why there modules are not released?
2. Please sort all spring related modules by alphabet order.

Reply:
Are these modules also not supported?

Reply:
These modules will be released by other team.